qwandaq/src/main/java/life/genny/qwandaq/managers/Manager.java

```diff
@@ -19,9 +17,6 @@
 public abstract class Manager {
 	protected static Jsonb jsonb = JsonbBuilder.create();
 
-	@Inject
-	protected Logger log;
```

Comment:
This stops all the logs coming up as coming from the Manager class. Sad but I will figure out another way to have a generic logger like this that shows the origin class